scan_org_sboms.py

```
@@ -6,203 +6,52 @@
 ORG = os.getenv("ORG") or (len(sys.argv) > 1 and sys.argv[1])
 TOKEN = os.getenv("GITHUB_TOKEN") or (len(sys.argv) > 2 and sys.argv[2])
 if not ORG or not TOKEN:
-    print("Usage: ORG=<org> GITHUB_TOKEN=<pat> python3 scan_org_sboms.py <org> <token>", file=sys.stderr); sys.exit(2)
+    print("Usage: ORG=<org> GITHUB_TOKEN=<pat> [SUSPICIOUS_FILE=<path>] python3 scan_org_sboms.py <org> <token> [<suspicious_file>]", file=sys.stderr); sys.exit(2)
```

Ruff (0.13.1)

Multiple statements on one line (semicolon)

(E702)